README.md

```diff
@@ -57,3 +59,33 @@ It could even be an automated task whenever you update your specs.
 ## 5. Extensible
 The **Event Gateway** can extend its functionality via middlewares written by the community.
 A catalog of middlewares made by the community is also available.
+
+# Roadmap
+The idea is to keep iterating and support all the protocols AsyncAPI supports through [bindings](https://github.com/asyncapi/bindings).  
+However, we reduced the scope for the first versions, so we can give support to the most used protocols. 
+
+The list, which can be subject to change, is:
+
+- [Kafka](https://kafka.apache.org) - Under development.
+- [MQTT](https://mqtt.org)
+- [AMQP](https://www.amqp.org)
```

Comment:
Are these two under development, or just next on the list? Is it in that specific order or? I don't think this little list clarifies much to the end-user 😅

Reply:
I removed those mentions and specified that we aim to give support **only** to Kafka protocol for the first version.

Does it look better?

Reply:
👍